### PR TITLE
Upgrade Elixir & Phoenix

### PR DESCRIPTION
**Pull request by kitallis**
Followed this [guide](https://gist.github.com/chrismccord/bb1f8b136f5a9e4abc0bfc07b832257e).

* Elixir 1.10.2 -> 1.10.3
* Phoenix 1.4.16 -> 1.5.1

### CODE REVIEW

File: server/lib/pencil_space_server/application.ex [hunk 1]
```diff
@@ -10,6 +10,8 @@ defmodule PencilSpaceServer.Application do
     children = [
       # Start the Ecto repository
       PencilSpaceServer.Repo,
+      # Start the PubSub system
+      {Phoenix.PubSub, name: MyApp.PubSub},
```
Comment (yudistrange) [inline server/lib/pencil_space_server/application.ex]: The name of the `PubSub` should be `PencilSpace.PubSub` or something like that.

**Review by yudistrange — COMMENTED** [review]
Did you check the changes from here: https://www.phoenixdiff.org/?source=1.4.16&target=1.5.1#d2h-464955

**Comment by kitallis** [comment]
> Did you check the changes from here: https://www.phoenixdiff.org/?source=1.4.16&target=1.5.1#d2h-464955

Nope, but this is useful. Didn't find anything in particular that would break it though.

**Review by yudistrange — APPROVED** [review]
lgtm

**Pull request merged**